guides/image-sources/provisioning-installer-bootdisk-subnet.plantuml

```diff
@@ -0,0 +1,79 @@
+@startuml
+
+!include foreman.pstyle
+!$networkboot = 1
+
+title Installer-based provisioning with subnet boot disk on bare metal
+
+actor       User
+participant "Provisioned\nHost" as Host
+participant "Foreman" as Foreman
+participant "Foreman\nProxy" as Proxy
+participant DHCP
+participant DNS
+!if ($puppet)
+participant "Puppet\nserver" as Puppet
+!endif
+
+note over Host : Powered off
+
+== Create host in Foreman ==
+
+User -> Foreman : Create host
```

Comment:
I still think the host creation should be done after the subnet image generation,
emphasizing the fact that it's not a precondition for the subnet image.

A quick glance at the diagram might lead one to think that host creation is a precondition, when in fact it's not.